### PR TITLE
[MINOR] Add Jenkins and AppVeyor badges at README.md

### PR DESCRIPTION
**Pull request by HyukjinKwon**
## What changes were proposed in this pull request?

This PR adds Jenkins build and AppVeyor build badges

Take a look at https://github.com/apache/spark/blob/cc24b9e0b35e947e97b47ca9d1b350bb37148b41/README.md

## How was this patch tested?

Manually tested by `grip`.

### CODE REVIEW

**Comment by HyukjinKwon** [comment]
option 1.

screen shot 2019-01-23 at 8 17 14 pm


option 2.

screen shot 2019-01-23 at 8 17 33 pm

**Comment by HyukjinKwon** [comment]
take a look here https://github.com/apache/spark/blob/fe3dc9caf7d36c701d227d9e0d401923f0e2f4fd/README.md

File: README.md [hunk 1]
```diff
@@ -1,4 +1,13 @@
-# Apache Spark
+[![Jenkins Build](https://amplab.cs.berkeley.edu/jenkins/job/spark-master-test-sbt-hadoop-2.7/badge/icon)](https://amplab.cs.berkeley.edu/jenkins/job/spark-master-test-sbt-hadoop-2.7)
+[![AppVeyor Build](https://img.shields.io/appveyor/ci/ApacheSoftwareFoundation/spark/master.svg?style=plastic&logo=appveyor)](https://ci.appveyor.com/project/ApacheSoftwareFoundation/spark)
+[![PyPI Downloads](https://img.shields.io/pypi/dm/pyspark.svg?label=pypi%20downloads&style=plastic)](https://pypi.org/project/pyspark)
+[![Commit Activity](https://img.shields.io/github/commit-activity/4w/apache/spark.svg?style=plastic)](https://github.com/apache/spark/commits/master)
```
Comment (HyukjinKwon) [inline README.md]: Btw I think we should keep the badges less then 5 if we're going ahead for those badges. Otherwise looks a bit messy imho.

**Comment by srowen** [comment]
There are a number of badges we could add; which ones might be useful to users or developers? I'm not sure. If one of the Jenkins jobs is running, if one of the Appveyor jobs is passing or not, is that useful info per se? Will number of commits matter much and PyPI downloads? I like the code-quality badges and Travis badge for small projects, to show that it's current and maintained, but those aren't relevant here. I think I might leave these off.

**Comment by HyukjinKwon** [comment]
But for both Jenkins and AppVeyor (I'm okay with getting rid of other badges), we might quickly know if the tests are globally being failed or not. But hmmm I don't feel strongly. I can take them out in a couple of days if there are no other feedback. How about the logo?

**Comment by srowen** [comment]
We have the amplab Jenkins for status, but I suppose this is another place you might notice. Is the AppVeyor test reliable enough to report here? I can see adding the build status to give a little more warning if something seems to be wrong.

The PyPI downloads and commits doesn't seem as useful.

**Comment by HyukjinKwon** [comment]
Yup, I'm going to take both out, and then gonna take the other out too if there are no more feedback about that. Was wondering how you guys feel about this.

**Comment by HyukjinKwon** [comment]
BTW, I wouldn't also just close this if there's not much feedback about the logo as well. It's not something worth if people are not sure or don't particularly prefer - it's just making something pretty.

**Comment by rxin** [comment]
I think the build ones are definitely useful. The rest I don't know how useful it is.

**Comment by HyukjinKwon** [comment]
Let me take out the logo too.

**Review by felixcheung — APPROVED** [review]
I like the build logo

**Comment by HyukjinKwon** [comment]
Just in case people might want to take the previous change .. take a look at https://github.com/apache/spark/blob/4938c14f4d23880b477d43a6d83ed7e427cb6c7f/README.md and let me know if anyone prefers this.

**Comment by felixcheung** [comment]
I kinda like the logo too, why not? :)
I don't feel strongly about it

**Comment by HyukjinKwon** [comment]
Yes .. :). WDYT @cloud-fan, @gatorsmile, @holdenk? Looks there are no particular opinion about that logo itself so far - take a look at https://github.com/apache/spark/blob/4938c14f4d23880b477d43a6d83ed7e427cb6c7f/README.md (I referred https://github.com/nodejs/node and https://github.com/twbs/bootstrap).

BTW, I expected a bit of forth and back since the preference matters here. Feel free to let me know if there's any preference.

**Comment by HyukjinKwon** [comment]
adding more people to ask preference @BryanCutler, @viirya, @ueshin, @rdblue, @MaxGekk, @mgaido91 who I have talked time to time (sorry if this cc was too arbitrary).

**Comment by mgaido91** [comment]
Thanks for pinging me @HyukjinKwon . Honestly I prefer the current option. It is a README, so the main goal is to provide general information about the project in the clearest possible way. It is not a marketing window where a logo may be more attractive.

**Comment by viirya** [comment]
I don't have strong option about this. One idea is currently two icons looks very similar (showing "build running" and "build passing" with different colors). I can't tell too much info from them (e.g., what difference between two).

**Comment by HyukjinKwon** [comment]
Right. Let me propose just build badges here for now. Let me merge this one in few days as is then.

**Comment by viirya** [comment]
> ^ if you take a look closely one has AppVeyor logo :D.

I didn't see it in IntelliJ IDEA's markdown visual editor. Saw it when manually opened it in browser. :)

**Review by BryanCutler — APPROVED** [review]

**Comment by HyukjinKwon** [comment]
Merged to master.

**Pull request closed without merging**